library/slack.py

```diff
@@ -13,16 +13,16 @@
 class Slack:
     users = {}
     user_names_to_emails = {
-        "keith": "keith@cognimate.ai",
-        "aditham": "prakash@cognimate.ai",
-        "mshash": "mithali@cognimate.ai",
-        "pradeepjavangula": "pradeep@cognimate.ai",
+        "arjun": "arjunkhanna9079@gmail.com",
+        "emma": "emmasmithcto6306@gmail.com",
+        "john": "johnbrowncpo8808@gmail.com",
+        "joy": "joytaylor1106@gmail.com",
     }
     user_ids_to_emails = {
-        "U06EP4STTQ8": "keith@cognimate.ai",
-        "U06E2R1AYCU": "prakash@cognimate.ai",
-        "U06J697E08Y": "mithali@cognimate.ai",
-        "U06DKQZ2J4F": "pradeep@cognimate.ai"
+        "U06E2R1AYCU": "arjunkhanna9079@gmail.com",
```

Comment:
The slack digest you have doesn't contain any reference to us, right? Ideally we add to these lists rather than replace. I think that can happen if

1. Your slack_response.json only refers to these users
2. You search replace all instances of each of these ids and make up new ones
3. You keep a list of the old and new ids in the user_ids_to_email map
4. The user names to emails map contains both as well.

Ideally these should live in some resource as well but let's save that for later.

Reply:
Yes, the Slack digest doesn't contain any reference to us. I created new data with the 4 new IDs and emails that I've mentioned here. Are you saying that I can keep both our names, emails, and IDs, along with the new ones, and the mapping would only be done with the new ones because my data only contains the new IDs?

Reply:
Yep

Reply:
Done.